README.md

```diff
@@ -160,13 +160,16 @@ Please see [Getting Started](./docs/getting-started.md) for an introduction to t
 
 Please see the documentation below for details about the ORT configuration.
 
-* [The .ort.yml file](./docs/config-file-ort-yml.md) - project-specific license finding curations, exclusions
+* The [ORT configuration](./model/src/main/resources/reference.conf) file - the main configuration file for the
+  operation of ORT. This configuration is usually maintained by a system administrator and created when ORT is taken
```

Comment:
I would not says "system administrator" but ORT administrator/manager as he former refers to IT system administrator

Reply:
Referring to an "IT system administrator" was actually my intention. In a corporate setup it probably indeed *is* an IT system administrator who's hosting ORT (the database etc.). But I also see how it can make sense to relax the wording and will make an update.